README.md

```diff
@@ -1 +1,225 @@
-# Exonum RFCs
\ No newline at end of file
+# Exonum RFCs
+[Exonum RFCs]: #exonum-rfcs
+
+Exonum RFC process is partially derived from [the Rust RFC process](https://github.com/rust-lang/rfcs/blob/master/README.md)
```

Comment:
According to this comment https://github.com/rust-lang/rfcs/issues/1259#issuecomment-220753590, we are not first.

More info on `rust-lang/rfcs` licensing:
https://github.com/rust-lang/rfcs/blob/master/text/2044-license-rfcs.md
https://github.com/rust-lang/rfcs/pull/2075